src/ucp/wireup/wireup.c

```diff
@@ -466,15 +474,12 @@ ucp_wireup_process_pre_request(ucp_worker_h worker, ucp_ep_h ep,
 
     ucs_assert(ucp_ep_get_cm_wireup_ep(ep) != NULL);
     ucs_assert(ep->flags & UCP_EP_FLAG_CONNECT_WAIT_PRE_REQ);
+    ucs_assert(ucp_ep_config(ep)->key.err_mode == msg->err_mode);
```

Comment:
IMO better use ucs_assertv or the new function to check err mode mismatch

Reply:
done